rocksdb/backup.pxd

```diff
@@ -30,6 +35,6 @@ cdef extern from "rocksdb/utilities/backupable_db.h" namespace "rocksdb":
         Status RestoreDBFromLatestBackup(string&, string&) nogil except+
 
     cdef Status BackupEngine_Open "rocksdb::BackupEngine::Open"(
-            Env*,
             BackupableDBOptions&,
+            Env*,
```

Comment:
I was very surprised at this change, as I thought they broke the API without bumping the major number. But it seems they kept the old signature for compatibility..

In any case, if I am reading correctly this new signature has been in the code since 6.10, so we should be fine with the change, right?

Reply:
Ah yes, I see you're right... both the new and deprecated signatures are in there. I thought I'd made this change out of necessity, but I'm no longer convinced.